com.unity.render-pipelines.universal/Runtime/Passes/DeferredPass.cs

```diff
@@ -28,6 +28,15 @@ public override void Configure(CommandBuffer cmd, RenderTextureDescriptor camera
         {
             RenderTargetIdentifier lightingAttachmentId = m_DeferredLights.GbufferAttachmentIdentifiers[m_DeferredLights.GBufferLightingIndex];
             RenderTargetIdentifier depthAttachmentId = m_DeferredLights.DepthAttachmentIdentifier;
+            if (m_DeferredLights.UseRenderPass)
+            {
+                var inputAttachments = new RenderTargetIdentifier[]
```

Comment:
This will cause allocation on heap and increase GC pressure.

Reply:
yes, already fixed this